Predictor_submission_template.ipynb

```diff
@@ -88,6 +96,8 @@
     "df = pd.read_csv(DATA_URL,\n",
     "                 parse_dates=['Date'],\n",
     "                 encoding=\"ISO-8859-1\",\n",
+    "                 dtype={\"RegionName\": str,\n",
```

Comment:
Removes a warning from Pandas complaining region columns have multiple types, because of the empty values.
You'll see this fix in a few places in this PR